trigger/coap/descriptor.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "flogo-coap",
+    "type": "flogo:trigger",
+    "version": "0.0.1",
+    "title": "Receive CoAP Message",
+    "description": "Simple CoAP Trigger",
+    "settings": [
+      {
+        "name": "port",
+        "type": "integer",
+        "required": true,
+        "description": "The port number"
+      }
+    ],
+    "output": [
+      {
+        "name": "queryParams",
+        "type": "params",
+        "description": "The query parameters"
+      },
+      {
+        "name": "payload",
+        "type": "string",
+        "description": "The payload of the CoAP Message"
+      }
+    ],
+    "handler": {
+      "settings": [
```

Comment:
I noticed in the activity you specify a method, if that is the case, shouldn't there be a method in the handler settings as well?

Reply:
I looked at couple of examples of CoAp server, they don't use method anywhere. So I removed it, tested it and it seems to work fine. But I would still take that with a grain of salt as I don't understand CoAp completely  .

Reply:
take a look at https://opensourceforu.com/2016/09/coap-get-started-with-iot-protocols/

I think the methods are need, because it is basically like REST over http.  Also if our activity uses them, the trigger should observe them as well.